### DISCUSSION TITLE
Issue about co pilot

### DISCUSSION THREAD

**Discussion by Oyindacodes2** [community/community#19300] [Copilot Conversations] [answered]
Is the co pilot going to be free forever?

**Comment by damoski2** [comment, marked as answer]
For now it is free until August 22nd, then you'll have to pay $10 monthly to have continual access and $300 for yearly plan

**Comment by thunder-007** [comment]
It's free for ever if you are a famous opensource contributor or student

**Comment by kskmail29** [comment]
I have a subscription for co-pilot and when I login from another computer to github it asks  to choose subscription. I would except the account should be enabled with copilot access
